priam/src/main/java/com/netflix/priam/backup/parallel/IncrementalConsumer.java

```diff
@@ -76,6 +79,9 @@ public Void retriableCall() throws Exception {
                         if (is == null) {
                             throw new NullPointerException("Unable to get handle on file: " + bp.getFileName());
                         }
+                        // Important context: this upload call typically has internal retries but those are only
```

Comment:
Filesystem upload does not have retries AFAIK. The only retry coming here is from RetryCallable used just above.